starknet_py/net/models/abi/parser.py

```diff
@@ -158,3 +171,15 @@ def _group_by_entry_name(
                 )
             grouped[name] = entry
         return grouped
+
+
+def _to_json(value):
+    class DataclassSupportingEncoder(json.JSONEncoder):
+        def default(self, o):
```

Comment:
Maybe rename `o` to `object`?

Reply:
This is the name in parent class. If this method is called with keyword argument it would fail.

Reply:
Oh, I see